core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -132,7 +132,7 @@ class KafkaApisTest extends Logging {
     clientControllerQuotaManager, replicaQuotaManager, replicaQuotaManager, replicaQuotaManager, None)
   private val fetchManager: FetchManager = mock(classOf[FetchManager])
   val sharePartitionManager : SharePartitionManager =
-    new SharePartitionManager(replicaManager, new SystemTime(),  new ShareSessionCache(1000, 100), 30000, 5, 200)
+    new SharePartitionManager(replicaManager, new SystemTime(),  new ShareSessionCache(1000, 100), 30000, 5, 200, "MockShareGroupPersisterClassPath")
```

Comment:
I think it's a class name, not a classpath.

Reply:
no, we need to enter the complete path if we want to use a class. Example - `org.apache.kafka.server.group.share.DefaultStatePersister`

Reply:
That is a fully qualified class name. A classpath is "/usr/lib/classes:/etc/whatever".

Reply:
okay. Anyways, realized it should be an empty string for `KafkaApisTest` to pass since the value will be used to initialize the `SharePartition`